### DISCUSSION TITLE
Tags on electricity processes for results calculation

### DISCUSSION THREAD

**Discussion by bl-young** [NETL-RIC/ElectricityLCI#262] [Ideas]
For practitioners, I wonder if tagging all electricity processes ([Tags](https://greendelta.github.io/openLCA2-manual/cheat/tags.html)) could help with results interpretation. With the current baseline I don't think there is a great way to assess the impacts from electricity generation broadly, since each generation process is distributed.

I'm not very familiar with running results this way, but I don't think it would be hard to implement within the schema.

**Comment by dt-woods** [comment]
Do you want to elaborate a little more on what you're thinking with 'help with results interpretation'? I see that "contribution by tags" is an option for results. Not sure what difference you'd see from contributions by process. Another concern is mixing library tags with user-defined tags, so tag naming should be mindful of that.

**Reply by bl-young** [reply]
If I'm building a model and using the electricity baseline in the background, when I run results, I will see the emissions from each process listed separately. In this case, that means emissions from each generator type, as well as fuel supply chains are all individual contributors to my total result. But in many cases, what people care about broadly are the cradle-to-gate impacts from electricity into their model, which in aggregate may be much greater than any individual source of emissions in the model (by process)
